src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java

```diff
@@ -11,6 +11,7 @@
 
     public final String id;
     public final String caseRef;
+    public final String previousServiceCaseRef;
```

Comment:
might be good to add a note here clarifying that it's not the 'previous ServiceCaseRef', it's the 'previous service caseRef'.

Reply:
or rename the field altogether

Reply:
This "previous service case reference" is a convention that is shared with Exela and CCD, so I'd rather not rename it. I'll add a comment.

Reply:
Maybe I should call it legacyId, as that's how we normally refer to it?

Reply:
`legacyCaseRef` it is